Resources/Prototypes/DeltaV/Roles/Jobs/Command/administrative_assistant.yml

```
@@ -32,6 +32,10 @@
   access:
     - Command
     - Maintenance
+    - Janitor
+    - Service
+    - Bar
+    - Kitchen
```

Changes to upstream files should include some clarifying comments, like this for example:
```suggestion
    - Janitor # Floof section - expanded access list
    - Service
    - Bar
    - Kitchen # Floof section end
```